var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -403,6 +404,7 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
         depends_on(kokkos_spec, when="@14.4.0 +kokkos {0}".format(arch_str))
 
     depends_on("adios2", when="+adios2")
+    depends_on("binder@1.3:", when="@15: +python", type="build")
     depends_on("binder@1.3:", when="+pytrilinos2", type="build")
```

Comment:
Duplicate in here now, remove this and all of the others that have when='+pytrilinos2'